sycl/plugins/esimd_cpu/pi_esimd_cpu.cpp

```diff
@@ -268,9 +417,10 @@ template <class KernelType, class KernelArgType, int DIMS> class libCMBatch {
 
     SpaceDim[0] = (uint32_t)Range[0];
 
-    PLACEHOLDER_UNUSED(WrappedLambda_ID_1DIM);
-    // TODO : Invoke invokeLambda_ID_1DIM through CM's multi-threaded
-    // kernel launching with WrappedLambda_ID_1DIM and dimension info
+    ESimdCPUKernel ESimdCPU((fptrVoid)InvokeLambda_ID_1DIM, GroupDim, SpaceDim);
```

Comment:
Are UnusedID and UnusedRange actual arguments really unused? If so why not just remove them?

Reply:
I'll remove `UnusedID/Range` as they are no longer used.

Reply:
`UnusedID` and `UnusedRange` are removed.